packages/insured-bridge-relayer/src/index.ts

```diff
@@ -35,7 +35,7 @@ export async function run(logger: winston.Logger, l1Web3: Web3): Promise<void> {
 
     // The logger is having issues with logging nested BNs. Remove this for now. Is indirectly fixed in PR https://github.com/UMAprotocol/protocol/pull/3656
     /* eslint-disable @typescript-eslint/no-unused-vars */
-    const { rateModels, ...logableConfig } = config;
+    const { ...logableConfig } = config;
```

Comment:
seeing you've removed this you can ditch this thing in totality. you can remove the comment above, and change the `log` below to include `config`. the whole thing is now `logable`.